src/MongoDB.Driver.Core.Dotnet/MongoDB.Driver.Core.Dotnet.csproj

```diff
@@ -18,6 +18,10 @@
     <Compile Include="..\MongoDB.Driver.Core\**\*.cs;..\MongoDB.Shared\ApplicationNameHelper.cs;..\MongoDB.Shared\Hasher.cs;..\MongoDB.Shared\MaxTimeHelper.cs" Exclude="bin\**;obj\**;**\*.xproj;packages\**" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="System.Diagnostics.TextWriterTraceListener" Version="4.0.0" />
```

Comment:
We can't use 4.3 since that would require updating TraceSource to 4.3.
This will cease to be a problem once we switch over to .NET Standard 2.0.

Reply:
Additionally. the .NET Core test projects already had a dependency on `TextWriterTraceListener` 4.0.0